src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/app.config

```diff
@@ -18,6 +18,14 @@
         <assemblyIdentity name="Newtonsoft.Json" publicKeyToken="30ad4fe6b2a6aeed" culture="neutral" />
         <bindingRedirect oldVersion="0.0.0.0-9.0.0.0" newVersion="9.0.0.0" />
       </dependentAssembly>
+      <dependentAssembly>
```

Comment:
These are usually explicitly removed as the redirects are known to cause issues with NGEN. In script, downgrade in the redirects to avoid issues. We need to be cautious and perform a full NGEN validation if we're touching them.

Reply:
System.Diagnostics.DiagnosticSource should be okay -- we don't downgrade that one: https://github.com/Azure/azure-functions-host/blob/v1.x/src/WebJobs.Script.WebHost/Web.config#L237-L240

System.Net.Http, though, we do downgrade -- https://github.com/Azure/azure-functions-host/blob/v1.x/src/WebJobs.Script.WebHost/Web.config#L229-L232. I'm guessing we can't use this version or our NGEN/cold-start will suffer. We'll need to see if anything breaks when downgrading. We've not run into anything yet...

Reply:
We don't currently downgrade `System.Diagnostics.DiagnosticSource`, but it was one of the dependencies that caused issues in the past, so we just need to be extra diligent with NGEN validation.